### PR TITLE
A bit of packaging

### PR DESCRIPTION
**Pull request by sebageek**
With servefile being a proper package, packaging it for pip will be a lot more fun.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Serve files from shell via a small HTTP server. The server redirects all HTTP
+requests to the file, so only IP and port must be given to another user to
+access the file. Its main purpose is to quickly send a file to users in your
+local network, independent of their current setup (OS/software). Besides that
+it also supports uploads, SSL, HTTP basic auth and directory listings.
+
+Features:
+ * serve single file
+ * serve a direcotry with directory index
```
Comment (hartwork) [inline README.md:12]: Typo `direcotry`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * file upload via webinterface
+ * https with on the fly generated self singed SSL certificates
+ * http basic authentication
```
Comment (hartwork) [inline README.md:15]: Typo `singed`, HTTP(s) would look better uppercase

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ * serving files/directories as on request generated tar files
+
+Install
+-------
+
+Via pip
+```shell
+pip install servefile
+```
+After installation either execute `servefile --help` or `pip -m servefile --help`
```
Comment (hartwork) [inline README.md:25]: `pip -m` should probably be `python -m`

File: setup.py [hunk 4]
```diff
@@ -1,42 +1,52 @@
...
-		'Topic :: Utilities',
-	],
+    name='servefile',
+    description='Serve files from shell via a small HTTP server',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    platforms='posix',
+    version='0.5.0',
+    license='GPLv3 or later',
+    url='https://seba-geek.de/stuff/servefile/',
```
Comment (hartwork) [inline setup.py:16]: No GitHub here?

**Review by hartwork — CHANGES_REQUESTED** [review]
Found a few tiny things:

**Comment by sebageek** [comment]
Already merged

**Pull request closed without merging**